Modules/_elementtree.c

```diff
@@ -257,6 +257,7 @@ typedef struct {
 } ElementObject;
 
 
+#define _Element_CAST(PTR) ((ElementObject *)(PTR))
```

Comment:
In Python, the convention is to use an "object pointer", `op`:

```suggestion
#define _Element_CAST(op) ((ElementObject *)(op))
```

Same remark for _ElementIter_CAST() below.

Reply:
AH! "op" stands for "object pointer". I always thought that "op" was for "operand".